### PR TITLE
Add SOCRBridge and disable SOCtoPSDBridge

### PR DESCRIPTION
**Pull request by blegat**
Closes https://github.com/JuliaOpt/MathOptInterface.jl/issues/637

### CODE REVIEW

File: src/Bridges/socr_bridge.jl [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                              G::Type{<:MOI.AbstractVectorFunction},
+                              ::Type{MOI.SecondOrderCone}) where T
+    S = MOIU.promote_operation(/, T, MOIU.scalar_type(G), T)
+    Y = MOIU.promote_operation(-, T, S, S)
+    Z = MOIU.promote_operation(+, T, S, S)
+    F = MOIU.promote_operation(vcat, T, Z, Y, G)
+    return SOCRBridge{T, F, G}
+end
+
+# Attributes, Bridge acting as an model
```
Comment (frapac) [inline src/Bridges/socr_bridge.jl]: Is there a typo: `an` -> `a model`?

**Comment by blegat** [comment]
Currently blocked by https://github.com/JuliaLang/julia/issues/32167

**Comment by chriscoey** [comment]
So RSOC and SOCR are different bridges. It's not clear from the naming which one does which. Are there slightly more verbose names we could use? If @mlubin is ok with the naming then don't worry.

**Comment by mlubin** [comment]
Neither RSOC nor SOCR are standard abbreviations so we should try to avoid using either. RotatedToSOC and SOCToRotated are alternatives.

**Comment by blegat** [comment]
I moved the naming to a separate issue as it would be breaking:
https://github.com/JuliaOpt/MathOptInterface.jl/issues/860
Any other objection ? The failure is now resolved thanks to https://github.com/JuliaOpt/MathOptInterface.jl/pull/834

**Pull request merged**